src/libsystemd/sd-bus/bus-common-errors.c

```diff
@@ -101,6 +101,8 @@ BUS_ERROR_MAP_ELF_REGISTER const sd_bus_error_map bus_common_errors[] = {
         SD_BUS_ERROR_MAP(BUS_ERROR_TRANSFER_IN_PROGRESS,         EBUSY),
 
         SD_BUS_ERROR_MAP(BUS_ERROR_NO_PRODUCT_UUID,              EOPNOTSUPP),
+        SD_BUS_ERROR_MAP(BUS_ERROR_NO_PRIVILEGE_TO_UPDATE_FILE,  EACCES),
```

Comment:
hmm, so i was wondering: why not use SD_BUS_ERROR_ACCESS_DENIED here?

i figure because you want to communicate that it's not the client that has trouble accessing hostnamed, but hostnamed having access trouble with the file it changes. Which is probably a sensible distinction to make. But I wonder if we should maybe put even more emphasis on the file as being the culprit. i.e. BUS_ERROR_FILE_IS_PROTECTED or so? i.e. put the blame clearly on the file?